lib/bemhtml/index.js

```diff
@@ -158,7 +158,7 @@ BEMHTML.prototype.renderClose = function renderClose(prefix,
 
   // NOTE: maybe we need to make an array for quicker serialization
   attrs = utils.extend(attrs, ctx.attrs);
-  if (attrs) {
+  if (attrs && Object.prototype.toString.call(attrs) === '[object Object]') {
```

Comment:
Because you already did `Object.prototype.toString.call(attrs) === '[object Object]'` instead of `typeof attrs === 'object'`